### PR TITLE
Address management

### PR DESCRIPTION
**Pull request by AtulPant2704**
- Add, remove and update address

### CODE REVIEW

File: src/pages/Profile/components/Address/AddressModal.jsx [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+        setAddress({ ...address, [name]: value });
+    };
+
+    const checkInputs = () => {
+        return address.name && address.street && address.city && address.state && address.country && address.zipCode && address.mobile;
+    }
+
+    const callAddNewAddressHandler = () => {
+        if (checkInputs()) {
+            console.log("here");
```
Comment (Abhijeet43) [inline src/pages/Profile/components/Address/AddressModal.jsx]: - Get rid of console.log

**Review by Abhijeet43 — APPROVED** [review]
- The functionality is working as intended
- Suggestions: 
  - Can use numeric input field for zip-code as they are majorly numeric in nature
  - There is no way to login directly unless redirected in mobile view you can add that for good UX

**Comment by AtulPant2704** [comment]
> * The functionality is working as intended
> * Suggestions:
>   
>   * Can use numeric input field for zip-code as they are majorly numeric in nature
>   * There is no way to login directly unless redirected in mobile view you can add that for good UX

Thanks for suggestinh such great UX change, never thought about it.

**Review by AnkurChunekar — APPROVED** [review]
Other than the changes suggested by  @Abhijeet43 
- everything looks good to merge
- The functionality is working as intended.
- code is neat and clean

**Pull request merged**